internal/ctl/app/version.go

```diff
@@ -0,0 +1,106 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/yaml"
+
+	"github.com/envoyproxy/gateway/internal/cmd/version"
+	"github.com/envoyproxy/gateway/internal/ctl/options"
+	"github.com/envoyproxy/gateway/internal/utils/kube"
+)
+
+func NewVersionsCommand() *cobra.Command {
+	var (
+		output string
+		remote bool
+	)
+
+	versionCommand := &cobra.Command{
+		Use:     "versions",
+		Aliases: []string{"version"},
+		Short:   "Show versions",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return versions(cmd.OutOrStdout(), remote, output)
+		},
+	}
+
+	flags := versionCommand.Flags()
+	options.AddKubeConfigFlags(flags)
+	versionCommand.PersistentFlags().BoolVar(&remote, "remote", false, "if true, retrieve server version from cluster")
+	versionCommand.PersistentFlags().StringVarP(&output, "output", "o", "json", "One of 'yaml' or 'json'")
+
+	return versionCommand
+}
+
+type VersionInfo struct {
+	ClientVersion  string                   `json:"client"`
+	ServerVersions map[string]*version.Info `json:"servers,omitempty"`
```

Comment:
`ServerVersions` should be updated if the `remote` flag is removed and `egctl version` always returns the server version.